go/libkb/device_clone_token.go

```diff
@@ -0,0 +1,45 @@
+// Copyright 2018 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+package libkb
+
+import "fmt"
+
+const (
+	DefaultCloneTokenValue string = "00000000000000000000000000000000"
+)
+
+type DeviceCloneToken struct {
+	Prior  string
+	Stage  string
+	Clones int
+}
+
+func deviceCloneTokenPaths(un NormalizedUsername) (p, s, c string) {
```

Comment:
Is it possible to store these things in LevelDB?  It'd prefer that over a whole new set of configuration files.  It's my understanding that the protocol should work even if these tokens are blasted away (since it's usually OK to blast away LevelDB).